modules/models/src/main/java/org/eclipse/xpanse/modules/models/policy/userpolicy/UserPolicyUpdateRequest.java

```diff
@@ -21,7 +20,6 @@ public class UserPolicyUpdateRequest {
     /**
      * The id of the policy.
      */
-    @NotNull
```

Comment:
why is this removed?

Reply:
![image](https://github.com/eclipse-xpanse/xpanse/assets/122504203/d3e29786-8721-4454-9dd9-53b3d1ad3bc0)
The API interface has changed and the id has been moved to the path, so this field can be null.